clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -100,7 +90,8 @@ class AzureResourceManagerClient extends AzureBaseClient {
     ResourceGroupListParameters parameters = new ResourceGroupListParameters()
```

Comment:
We should remove the getResourcesGroupsForApp since there is a 1:1 correlation now between an app and a resource group. This method wont find you the server groups or clusters.